06week/Checkpoint2

```diff
@@ -0,0 +1,126 @@
+
+// Given 1000 digits of PI as strings, return an array of the digits as numbers
+
+const nums =[];
+for (let i in strNums) {
+  const n = parseInt(strNums[i]);
+  nums.push(n);
+}
+console.log(nums)
+
+
+// Find the sum of the even values
+
+let sumEvens=0;
+nums.forEach(num =>{
+if(num %2 ===0){
+ sumEvens = sumEvens + num;
+}
+});
+
+console.log(sumEvens);
+
+
+// define the index of the first "Austin" value (the value plus its index equals 512 area Code)
+
+const atxIdx = nums.findIndex((num, idx) => {
```

Comment:
careful with compatibility.  As we begin approaching the browser, this one isn't compatible with all browsers